CONTRIBUTING.md

```diff
@@ -226,8 +226,8 @@ maintainer.
 
 ## Initial setup
 
-1. Clone this repository with `git clone https://github.com/getporter/porter.git ~/go/src/get.porter.sh/porter`. Porter relies on being in the GOPATH.
-1. Run `make build install` from within the newly cloned repository.
+1. Clone this repository with `git clone https://github.com/getporter/porter.git ~/go/src/get.porter.sh/porter`.
+2. Run `make build install` from within the newly cloned repository.
```

Comment:
You don't need to change this back but I wanted to share why you will see numbered markdown lists with every item at `1.`. The markdown processor handles figuring out the number for you which is helpful when inserting new steps in the middle. You don't need to then update all the other numbers to match.

Reply:
Ah that was handy! Will keep that in mind for future changes 🙂